core/core-model/src/main/java/com/hbc/pms/core/model/AlarmCondition.java

```diff
@@ -0,0 +1,39 @@
+package com.hbc.pms.core.model;
+
+import com.hbc.pms.core.model.enums.AlarmSeverityEnum;
+import lombok.*;

```

Comment:
Please avoid wildcard imports from now on. Update the setting as following
![image](https://github.com/f-pms/API/assets/52276566/6fb0e175-1f39-48c7-aff4-0c8dc37d2eb7)
cc: @thinhle19 / @zKirinz

Reply:
Please ignore it until the check-style pipeline is completed, cc @thinhle19 @zKirinz